code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -0,0 +1,96 @@
+// /obj/item signals
```

Comment:
nitpick - for all of the dcs files, it's best to have signals be like this
```
/// text
#define
```

or atleast like this
`#define ... //! text`

this way langserv comments it

not really too important but just as a note

Reply:
ah, my bad, if you could tell i was reorganizing stuff, moving the object/item stuff into its own file